README.md

```diff
@@ -49,15 +49,15 @@ const signature = signWithPrivateKey(signMessage, privateKey)
 import { signWithPrivateKey } from "@lunie/cosmos-keys"
 import Cosmos from "@lunie/cosmos-js"
 
-const privateKey = Buffer.from(...)
-const publicKey = Buffer.from(...)
+const privateKey = Buffer.from(...);

```

Comment:
';' is not necessary. For consistency can we leave them out?

Reply:
should fail the linter, no?

Reply:
this is not a js file

Reply:
https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

Reply:
> https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

This is the google style guide and not a W3C spec. We use Standard-JS as our style guide (https://standardjs.com/rules-en.html#semicolons).
I also recommend reading "Mario Kondo - Magic Cleaning" and what she says about getting rid of things "don't spark joy". :)